test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -299,6 +299,21 @@ def test_r2plus1d_18_video(self):
         x = torch.randn(1, 3, 4, 112, 112, requires_grad=True)
         self.run_test(model, (x,), rtol=1e-3, atol=1e-5)
 
+    def test_fuse_conv_bn(self):
```

Comment:
Does the pass work for 1D or 3D ops? Can we add tests?

Reply:
for 3D ops there is a small mismatch

Reply:
Please check why. Test with a bigger model by adding more BN+Conv nodes.